gatsby/src/components/BackToTop/BackToTop.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState, useCallback } from "react";
+import ChevronUp from "@nice-digital/icons/lib/ChevronUp";
+import throttle from "lodash/throttle";
```

Comment:
Maybe we could install [lodash.throttle](https://www.npmjs.com/package/lodash.throttle) directly?

Reply:
(smaller dependency is always good)